webcurator-core/src/main/java/org/archive/io/CDXIndexer.java

```diff
@@ -1,21 +1,30 @@
 package org.archive.io;
 
-import java.io.File;
-import java.io.IOException;
-
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.netpreserve.jwarc.*;
+import org.netpreserve.jwarc.cdx.CdxFormat;
```

Comment:
Does this library need to be explicitly imported if jwarc.* is in the import above?

Reply:
Yeah, the asterisk only covers classes directly inside the jwarc namespace, so it doesn't recursively include sub-packages